build.sh

```diff
@@ -217,20 +218,25 @@ fi
 ########
```

Comment:
This line onwards to line 240 can be removed

Reply:
Oh sorry! Didn't not push properly...